pkg/api/validate/vm.go

```diff
@@ -79,7 +79,7 @@ var SupportedMasterVmSizes = map[api.VMSize]api.VMSizeStruct{
 	api.VMSizeStandardM128ms: api.VMSizeStandardM128msStruct,
 }
 
-var supportedWorkerVmSizes = map[api.VMSize]api.VMSizeStruct{
+var SupportedWorkerVmSizes = map[api.VMSize]api.VMSizeStruct{
```

Comment:
Comments about those two things would probably be helpful in-code I think.

Reply:
I am not entirely sure that making the variable public was the best solution tbh (starting with #2726), because now that becomes modifiable outside of the package. A simple public function would be safer than a global variable, since it would prefer accidental modification of the map. Probably something to look into in a follow up PR